### DISCUSSION TITLE
UX: Moving the mouse on the UI may cause RSI issues

### DISCUSSION THREAD

**Discussion by Thorwegian** [photoprism/photoprism#4263] [Ideas]
#### 1. What is not working as documented?

The desktop web version of the UI isn't very optimal when using a mouse. You often have to click many photos individually instead of drag selecting them, and picking an action requires a lot of mouse movement because there's no pop-up menu. This gave me repetitive strain injury problems the other day, after sorting thousands of pictures into albums, and I was unable to do my usual gym routine.

#### 2. How can we reproduce it?

1. Select a large number of photos.
2. Move them to an album.
3. Repeat for hours.
4. Get wrist pain.

#### 3. What behavior do you expect?

Reduce how much the mouse has to be moved to perform operations.

#### 4. What could be the cause of your problem?

Having thousands of photos to categorise upon initial import and wanting to get through a lot of them in a short time.

#### 6. Which software versions do you use?

(a) PhotoPrism Architecture & Build Number: AMD64, ARM64, ARMv7,...

x64
[PhotoPrism® CE](http://192.168.0.4:2342/library/about)
Build [240420-ef5f14bc4]

**Comment by graciousgrey** [comment]
There is no drag selection at this time however you can select multiple photos at once using shift click.

https://docs.photoprism.app/user-guide/navigate/#selection-mode-and-multi-select

**Reply by Thorwegian** [reply]
Thank you. This is not the same as a drag selection though. The issue of long movements between different parts of the screen also remains unaddressed. I see that this issue ticket has been refiled as an "idea", and I disagree with that line of thinking. As an experienced programmer myself, I see how this flaw of thinking occurs. One believes that a difficulty of using the software is cosmetic. But the entire idea of software is to serve a person.

**Reply by Thorwegian** [reply]
"User friendly", a.k.a. the computer is actually working.
